src/realm-settings/LocalizationTab.tsx

```diff
@@ -87,6 +103,34 @@ export const LocalizationTab = ({
     setAddMessageBundleModalOpen(!addMessageBundleModalOpen);
   };
 
+  const options = [
+    <SelectGroup label={t("defaultLocale")} key="group1">
+      {watchSupportedLocales
+        .filter((item) => item === realm?.defaultLocale)
+        .map((locale: string, idx: number) => (
+          <SelectOption key={`supported-locale-${idx}`} value={locale}>
+            {t(`allSupportedLocales.${locale}`)}
+          </SelectOption>
+        ))}
+    </SelectGroup>,
+    <Divider key="divider" />,
+    <SelectGroup label={t("supportedLocales")} key="group2">
+      {watchSupportedLocales
+        .filter((item) => item !== realm?.defaultLocale)
+        .map((locale: string, idx: number) => (
```

Comment:
These types are already known, so there is no need to specify them.
```suggestion
        .map((locale, idx) => (
```